### DISCUSSION TITLE
The Plugin used in the demo

### DISCUSSION THREAD

**Discussion by sleepymalc** [svsool/vscode-memo#583] [Q&A]
I'm wondering what plugin does the author used in the demo?
![image](https://user-images.githubusercontent.com/60427058/181466089-110ebb73-bafd-47d1-a1aa-aa0015a256e7.png)
More specifically, how can that code block (showing the file structure) be rendered directly in the editing panel as if it's in the preview? Need help!!!!

**Comment by svsool** [comment]
Hi @sleepymalc, I used markdown ```code blocks``` and [unix tree](https://en.wikipedia.org/wiki/Tree_(command)), this view doesn't use any plugins except Memo and built-in markdown extension.

**Reply by sleepymalc** [reply]
Yes, I understand that's something like ` ```code ``` ` syntax. But with basic markdown extension, my edit panel looks like this:
<img width="299" alt="image" src="https://user-images.githubusercontent.com/60427058/181886204-339dc71b-42d3-4e7b-9cf8-698b93a66b1d.png">
while in the preview I get 
<img width="236" alt="image" src="https://user-images.githubusercontent.com/60427058/181886309-978408d2-5da1-446c-bd0c-276a8d0d65bb.png">
which is something like what you had in your edit panel. Or am I missing something?

**Reply by svsool** [reply]
This is how it looks in the edit panel for me:

![image](https://user-images.githubusercontent.com/1452141/181898465-fad19d49-27bc-4b41-83ee-cbe0e6b0623a.png)

I assume that you want something like live preview, but I'm not sure whether one exists in vscode.

**Reply by sleepymalc** [reply]
Oh ok, so what in the demo is actually a screenshot of preview, while that `[[Demo]]` along with suggestions are just an image? My bad 😥

**Reply by svsool** [reply]
> a screenshot of preview

yes, correct
